packages/design-system/src/components/tree/tree-node.tsx

```diff
@@ -417,7 +417,10 @@ export const TreeItemBody = <Data extends { id: string }>({
           style={{ left: (level - 1) * INDENT + ITEM_PADDING_LEFT }}
           // We don't want this trigger to be focusable
           tabIndex={-1}
-          onClick={(event) => onToggle(event.altKey)}
+          onClick={(event) => {
+            event.stopPropagation();
```

Comment:
Let's comment every stopPropagation and preventDefault as they are often used incorrectly or without any reason.
For example why preventDefault didn;t help?